spec/models/user_spec.rb

```diff
@@ -287,5 +289,9 @@
         expect(@merchant.top_3_customers_by_total_paid).to eq([@user_5, @user_3, @user_2])
       end
     end
+    it '.total_items' do
```

Comment:
There could be more to this test

Reply:
At first I was thinking that testing a percent to be 0, and not anything else isn’t the greatest test. 
Now that I look at it closer, I see you are testing “.percent_items_sold” elsewhere. Now I’m wondering: if this is the test for “.total_items” shouldn’t it be testing that method directly and not “.percent_items_sold”. 
Also, adding the % sign is probably something that could be handled in the view and not the model, in my opinion.